src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import BooksList from '../containers/BooksList';
+import BooksForm from '../containers/BooksForm';
+
+const App = () => (
+  <>
```

Comment:
- [ ] [OPTIONAL] Adding react fragment here is not necessary as children are already wrapped in `div`

Reply:
Hi, good job by wrapping children in div tag here, and it is part of project requirements 👍 . But what I said is that,  you can remove the fragment now as, all children are already wrapped in div.